shared/teams/add-members-wizard/index.stories.tsx

```diff
@@ -25,6 +26,10 @@ const load = () => {
       <AddFromWhere onContinue={Sb.action('onContinue')} teamID={fakeTeamID} newTeam={true} />
     ))
     .add('Enable contacts', () => <EnableContacts onClose={Sb.action('onClose')} />)
+
+  Sb.storiesOf('Teams/Add by email wizard', module).add('Basic', () => (
```

Comment:
Putting this `.add` on the existing `storiesOf` above will give your story the decorator it needs so the teamID->teamname mapping works.

Reply:
do you mean like:
```
.add('Enable contacts', () => <EnableContacts onClose={Sb.action('onClose')} />)
.add('Basic', () => <AddEmail teamID={fakeTeamID} errorMessage="" />)
```

Reply:
Yeah, with a more specific name than 'Basic'